src/include.am

```diff
@@ -2,47 +2,22 @@
 # included from Top Level Makefile.am
 # All paths should be given relative to the root
 
-BUILT_SOURCES+= ctaocrypt/src/fips.c
-BUILT_SOURCES+= ctaocrypt/src/fips_test.c
-BUILT_SOURCES+= wolfcrypt/src/async.c
-BUILT_SOURCES+= wolfcrypt/src/fips.c
-BUILT_SOURCES+= wolfcrypt/src/fips_test.c
-BUILT_SOURCES+= wolfcrypt/src/selftest.c
-BUILT_SOURCES+= wolfcrypt/src/wolfcrypt_first.c
-BUILT_SOURCES+= wolfcrypt/src/wolfcrypt_last.c
-
-MAINTAINERCLEANFILES+= ctaocrypt/src/fips.c
-MAINTAINERCLEANFILES+= ctaocrypt/src/fips_test.c
-MAINTAINERCLEANFILES+= wolfcrypt/src/async.c
-MAINTAINERCLEANFILES+= wolfcrypt/src/fips.c
-MAINTAINERCLEANFILES+= wolfcrypt/src/fips_test.c
-MAINTAINERCLEANFILES+= wolfcrypt/src/selftest.c
-MAINTAINERCLEANFILES+= wolfcrypt/src/wolfcrypt_first.c
-MAINTAINERCLEANFILES+= wolfcrypt/src/wolfcrypt_last.c
-
-ctaocrypt/src/fips.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
-
-ctaocrypt/src/fips_test.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
-
-wolfcrypt/src/async.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
-
-wolfcrypt/src/fips.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
-
-wolfcrypt/src/fips_test.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
-
-wolfcrypt/src/selftest.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
-
-wolfcrypt/src/wolfcrypt_first.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
-
-wolfcrypt/src/wolfcrypt_last.c: Makefile
-	@test -e $(srcdir)/$@ || touch $(srcdir)/$@
+FIPS_FILES =					\
+	ctaocrypt/src/fips.c			\
+	ctaocrypt/src/fips_test.c		\
+	wolfcrypt/src/async.c			\
+	wolfcrypt/src/fips.c			\
+	wolfcrypt/src/fips_test.c		\
+	wolfcrypt/src/selftest.c		\
+	wolfcrypt/src/wolfcrypt_first.c		\
+	wolfcrypt/src/wolfcrypt_last.c
+
+BUILT_SOURCES+= $(FIPS_FILES)
+
+MAINTAINERCLEANFILES+= $(FIPS_FILES)
+
+$(FIPS_FILES): Makefile
+	touch $(srcdir)/$(@:$(srcdir)/%=%)
```

Comment:
Please don't unconditionally `touch` the FIPS files.  This unnecessarily updates the write timestamp of existing files, and if those files are symlinks, it is writing to files in a different repo.

Reply:
@douzzer What issue does that cause for the other repo?

I like it better than running this rule each time if `Makefile` is newer than the fips files.